README.adoc

```diff
@@ -154,24 +152,22 @@ query/microprofile-config.properties
+Test the connection between the `query` service and the `system` service by pointing your browser to the http://localhost:9085/query/systems/localhost URL. The `query` service returns the message: `{"fail":"Failed to reach the client localhost."}`.
 
-// file 2
-system/server.xml
-[source, XML, linenums, role='code_column']
-----
-include::finish/system/src/main/liberty/config/server.xml[]
-----
+This failure occurs because the `system` service was started with the `development` profile, while the configuration properties defined in the [hotspot=system file=1]`query/src/main/resources/META-INF/microprofile-config.properties` file are configured for accessing the `testing` environment. 
 
+To resolve the issue, you'll create a configuration profile in the `query` service with the configuration properties necessary to access the `system` service in the `development` environment.
 
 // =================================================================================================
 == Creating the configuration profile at the property level
 
-The config profile feature in MicroProfile Config supplies configurations for different environments such as development, testing, and production while only a single profile is active. The active profile is specified by using the `mp.config.profile` property, which can be set in any of the https://openliberty.io/docs/latest/external-configuration.html#_default_configuration_sources[ConfigSources^] or at application startup. Once it is set, the corresponding set of configuration properties that are associated with the active profile are used. 
+MicroProfile Config's config profile feature allows for the supply of configurations for different environments, such as development, testing, and production, while only a single profile is active. The active profile is set using the `mp.config.profile` property, which can be set in any of the ConfigSources or at application startup. Once set, the corresponding configuration properties associated with the active profile are used. 
 
-Navigate to the `start` directory to create configuration profile for the `query` service at the property level.
+Creating configuration profiles at the property level is useful when only a few configuration properties need to be set differently for each DevOps stage, such as when you require different database connection settings in different environments. 
 
-You can provide configuration properties for the `development` environment at the property level by specifying them in the following format: `%development.<property name>=<value>`. 
+To provide a configuration property for a particular config profile, use the `%<mp.config.profile>.<property name> = <value>` syntax.
```

Comment:
i find this a bit confusing. At line 164, you refer to "`mp.config.profile` property..." and the example syntax here is 
"`%<mp.config.profile>.<property name> = <value>`". So I think you have "mp.config.profile" used in two different contexts and its easy to wonder whether they are supposed to be the same thing.

Is there another way to show the syntax? like maybe

`%<config_profile_id>.<property name> = <value>`

?

Reply:
you might state earlier in the guide that each config profile has a unique ID

Reply:
Edited the change requested in https://github.com/OpenLiberty/draft-guide-microprofile-config-profile/pull/14/commits/1fb7c4965bdfe9a65e7ada76a6b52ab381be86ea